src/core/server/saved_objects/service/lib/crypto/crypto_cli.ts

```diff
@@ -0,0 +1,87 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Any modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
no need to copy elasticsearch license head

Reply:
The license head is required for each file to pass the lint check. Will replace with the OpenSearch one

Reply:
there is already Apache one above.